### ISSUE TITLE
package restore doesn't install packages.

### ISSUE THREAD

**Issue by elestedt** [NuGet/Home#2432]
I'm trying to get NuGet 3.3.0 to restore packages on a Windows 2012 server which will be used as a build slave for Jenkins. But I've run into a problem.

NuGet downloads the packages (I can see them appear in the users NuGet cache) but it won't install them. Instead I get: 

```
Restoring NuGet package Hangfire.1.5.4.
WARNING: Unable to find version '1.5.4' of package 'Hangfire'.
Unable to find version '1.5.4' of package 'Hangfire'.
```

In the cache folder I can see that this specific version has been downloaded.
Just to be on the safe side, I added the `packageRestore` options as `True` in the config file.

**Comment by yishaigalatzer** [comment]
Can you try a couple of things:
1. Please run nuget.exe locals -clear all
2. Can you share your nuget.config, and exact command you are running?

**Comment by elestedt** [comment]
```
>nuget locals -clear all
nuget locals -clear all
Clearing NuGet HTTP cache: %APPDATA%\Local\NuGet\v3-cache
Clearing NuGet cache: %APPDATA%\Local\NuGet\Cache
Clearing NuGet global packages cache: %USERPROFILE%\.nuget\packages\
Cache cleared.

>nuget restore -DisableParallelProcessing
nuget restore
MSBuild auto-detection: using msbuild version '14.0' from 'C:\Program Files (x86)\MSBuild\14.0\bin'.
Feeds used:
  %APPDATA%\Local\NuGet\Cache
  https://api.nuget.org/v3/index.json

Restoring NuGet package Hangfire.1.5.4.
WARNING: Unable to find version '1.5.4' of package 'Hangfire'.
Unable to find version '1.5.4' of package 'Hangfire'.
```

As you can see I've disabled parallel processing of restores, and I've replaced some absolute paths above with the environment variable equivalent.
Packages seem to be downloaded into _%APPDATA%\Local\NuGet\Cache_, at least that's where the pkg and spec files end up.
And the NuGet.config located in _%APPDATA%/Roaming/NuGet_ looks like this

``` xml
Roaming/NuGet/NuGet.Config
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <config>
    <add key="http_proxy" value="http://localhost:8888" />
    <add key="https_proxy" value="http://localhost:8888" />
  </config>
  <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
  </packageSources>
  <activePackageSource>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
  </activePackageSource>
<configuration>
  <packageRestore>
    <add key="enabled" value="True" />
    <add key="automatic" value="True" />
  </packageRestore>
</configuration>
```

**Comment by elestedt** [comment]
So, after much digging and trying to run with the _2.4.0-rc_ version, I got this message:

```
Error downloading 'Common.Logging.3.3.1' from 'https://api.nuget.org/v3-flatcontainer/common.logging/3.3.1/common.logging.3.3.1.nupkg'.
This implementation is not part of the Windows Platform FIPS validated cryptographic algorithms.
```

So, searching for that on Google I found [this](https://blogs.msdn.microsoft.com/brijs/2010/08/10/issue-getting-this-implementation-is-not-part-of-the-windows-platform-fips-validated-cryptographic-algorithms-exception-while-building-outlook-vsto-add-in-in-vs-2010/) article. After disabling FIPS in the registry as described there, both the RC and 2.3.0 started working.

Perhaps adding some error message to the 2.3.x series for this?
